frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -252,11 +260,19 @@ export const OperatorHubSubscribeForm: React.FC<OperatorHubSubscribeFormProps> =
     selectedTargetNamespace,
   );
 
+  const tokenType = 'role ARN';
+  const tokenizedAuth = getQueryArgument('tokenizedAuth');
```

Comment:
Can add `tokenizedAuth` to line 89 (and update lines 93 and 99 to use it as well, which several of us missed when @sg00dwin made those changes).

Reply:
sorry not sure I understand what the updates to 93 and 99 would look like